README.md

```diff
@@ -138,6 +138,70 @@ object Main extends App {
 }
 ```
 
+##### Basic Usage Streaming
+
+Assuming you have already created a `FaunaClient`.
+
+###### Flow subscriber
+
+It is possible to use the `java.util.concurrent.Flow` API directly by binding a `Subscriber` manually.
+
+In the example below we are capturing the 4 first messages.
+
+```scala
+client.stream(docRef).flatMap { publisher =>
```

Comment:
Perhaps include a comment that indicates how `docRef` is defined? Perhaps something like this:

```
// docRef is a reference to the document for which we want to stream updates.
// You can acquire a document reference with a query like the following, but it
// needs to work with the documents that you have.
// val docRef = client.query(Get(Ref(Collection("scoreboards"), "123")))
```